movie/src/components/MainContainer/index.jsx

```diff
@@ -24,17 +24,19 @@ const MainContainer = () => {
   };
 
   return (
-    <div className="actionAndCardContainer">
-      <ActionContainer movies={movies} onSearch={onGetMoviesAfterDoAction} />
-      <CardContainer
-        itemForCardContainer={{
-          items: movieList.slice(START_INDEX, fourMoviesLength),
-          fourItems: fourMoviesLength,
-          amountItemsDisplay: amountMoviesDisplay,
-        }}
-        onClick={onLoadMore}
-      />
-    </div>
+    <>
+      <div className="actionAndCardContainer">
+        <ActionContainer movies={movies} onSearch={onGetMoviesAfterDoAction} />
+        <CardContainer
+          itemForCardContainer={{
+            items: movieList.slice(START_INDEX, fourMoviesLength),
+            fourItems: fourMoviesLength,
```

Comment:
what is `fourItems`?

Reply:
> what is `fourItems`?
fourItems is refered to fourMoviesLength that is number (4 items or 4 movie)

Reply:
fourItems is refered to fourMoviesLength that is number (4 items or 4 movie)

Reply:
hmm. then could you find a clearer name for it. since the one you're using now is still quite confusing